Dockerfile

```
@@ -1,69 +1,69 @@
+COPY tests ./tests
+
+RUN cargo test --release --workspace
+RUN touch nexum_cli/src/main.rs && cargo build --release --workspace --exclude tests
+
+
+##################################################
+#################### STAGE 2 #####################
+##################################################
+FROM debian:bookworm-slim
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)